### PR TITLE
Improved wording

### PR DESCRIPTION
**Pull request by udidahan**
> Person should ensure that this message was not processed or was processed incorrectly before using retry feature.

This sounds weird.

### CODE REVIEW

File: servicepulse/intro-pending-retries.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The information about the message such as Failure timestamp, endpoint, stack tra
 
 ### Message Retry
 
-WARNING: Failed messages that are currently in the pending status can be retried again, however this feature should be used with care. Retrying pending messages can cause the endpoint to reprocess the same message multiple times. Person should ensure that this message was not processed or was processed incorrectly before using retry feature.
+WARNING: Failed messages that are currently in the pending status can be retried again, however this feature should be used with care. Retrying pending messages can cause the endpoint to reprocess the same message multiple times. Make sure that a given message was not processed, or was processed incorrectly, before using the retry feature.
```
Comment (SimonCropp) [inline servicepulse/intro-pending-retries.md]: Still struggling with the wording. How about...

> Prior to retrying a message ensure the message is not, either successfully or unsuccessful, processed.

Comment (udidahan) [inline servicepulse/intro-pending-retries.md]: Maybe it should be inverted:

> Do not retry a message if it was successfully processed

I'm not sure how to incorporate "not unsuccessful", "successful", and "incorrectly"

Comment (SimonCropp) [inline servicepulse/intro-pending-retries.md]: @WojcikMike can u chime in?

Comment (indualagarsamy) [inline servicepulse/intro-pending-retries.md]: @WojcikMike @udidahan @SimonCropp - how about:

WARNING: Failed messages that are currently in the pending status can be retried again, however this feature should be used with care. Retrying pending messages can cause the endpoint to reprocess the same message multiple times. Before using the retry feature, ensure that the same message is not already in the endpoint's queue or in the error queue.

Comment (WojcikMike) [inline servicepulse/intro-pending-retries.md]: @indualagarsamy I like your suggestion

**Pull request merged**